### PR TITLE
Update KDE Framework to 5.36

### PR DESCRIPTION
**Pull request by yopito**

### CODE REVIEW

**Comment by jnbr** [comment]
Current version is now 5.34.
KDE Framework should be updated soon as the currently used version 5.29 is not available on the download.kde.org server anymore.

**Comment by ebfe** [comment]
Also: http://www.openwall.com/lists/oss-security/2017/05/10/3

**Comment by yopito** [comment]
Thanks for the pointer. I've checked : 5.34 was not available 3 hours ago ;)
FYI, current status is: all these 5.33 packages build fine natively. I'm getting stuck on cross-building as you can see with my numerous WIP commits.
I will move on ASAP on 5.34

**Comment by yopito** [comment]
My packaging status (5.34):
* (native) build is fine with x86_64/glibc for all these packages
* cross build is fine for x86_64/musl for all these packages
* armV7 or aarch64 cross build status: **54**/75 packages are OK except:
    * `kjs`: might be fixable
    * `kjsembed` : depend on `kjs`
    * `kio`:  `kcoreaddons` cmake cross-build's support has to be fixed for `desktoptojson` (nightmare)
    * packages that depend on `kio`: `kdeclarative kparts kinit kdewebkit plasma-framework kded kdesignerplugin knewstuff kxmlrpcclient ktexteditor krunner kross knotifyconfig kmediaplayer khtml kdelibs4support kcmutils frameworkintegration`

**Comment by yopito** [comment]
My packaging status (5.34):
* (native) build is fine with x86_64-glibc for all these packages
* cross build is fine for x86_64-musl for all these packages
* armv7hf or aarch64 (glibc or musl) cross build status: **68**/75 packages are OK.  
* (still) broken packages with cross-build:
    * `kconfig` : wrong binary arch for `kconfig_compiler_kf5` (from `kconfig`)
    * `kdeclarative` : wrong binary arch for `kconfig_compiler_kf5` (from `kconfig`)
    * `plasma-framework` : wrong binary arch for `kconfig_compiler_kf5` (from `kconfig`)
    * `ktexteditor` : wrong binary arch for `kauth-policy-gen` (from `kauth`)
    * `krunner` : depend on `plasma-framework`
    * `kdelibs4support` : weird about `Qt5DesignerConfig.cmake` (from `qt5-tools-devel`)
    * `kcmutils` : depend on `kdeclarative`

**Comment by yopito** [comment]
Packaging status (5.34):
* (native) build is fine with x86_64-glibc for all these packages
* cross build is fine for x86_64-musl for all these packages
* armv7hf or aarch64 (glibc or musl) cross build status: **71**/75 packages are OK.  
* (still) broken packages with cross-build:
    * `plasma-framework` : wrong about `Qt5::Gui_GLESv2`
    * `krunner` : depend on `plasma-framework`
    * `ktexteditor` : wrong binary arch for `kauth-policy-gen` (from `kauth`)
    * `kdelibs4support` : weird about `Qt5DesignerConfig.cmake` (from `qt5-tools-devel`)

**Comment by yopito** [comment]
Packaging status (5.34):
* (native) build is fine with x86_64-glibc for all these packages
* cross build is fine for x86_64-musl for all these packages
* armv7hf or aarch64 (glibc or musl) cross build status: **72**/75 packages are OK.  
* (still) broken packages with cross-build:
    * `plasma-framework` : "target `Qt5::Gui_GLESv2` not found"
    * `krunner` : depend on `plasma-framework`
    * `kdelibs4support` : broken because of #6658
* version 5.35 is close to become available

**Comment by ebfe** [comment]
5.35 is out now. I don't think failing cross builds should prevent merging this.

**Comment by yopito** [comment]
Thanks for your support. I have most of pieces for 5.35, but i'm on travel. I will provide it ASAP next week.

Le 29 juin 2017 13:36:33 GMT+02:00, Michael Gehring <notifications@github.com> a écrit :
>5.35 is out now. I don't think failing cross builds should prevent
>merging this.
>
>-- 
>You are receiving this because you authored the thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/voidlinux/void-packages/pull/6052#issuecomment-311940766

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by yopito** [comment]
Ready for review and merge.

My packaging status (5.35):
* (native) build is fine with x86_64-glibc for all these packages
* cross build is fine for x86_64-musl for all these packages
* armv7hf or aarch64 (glibc or musl) cross build status: **72**/75 packages are OK.  
* broken cross-build packages (flagged with `broken` keyword) are:
    * `plasma-framework` : "target `Qt5::Gui_GLESv2` not found"
    * `krunner` : depend on `plasma-framework`
    * `kdelibs4support` : broken because of #6658

File: common/shlibs [hunk 1]
```diff
@@ -2589,18 +2591,22 @@ libKF5JobWidgets.so.5 kjobwidgets-5.26.0_1
 libKF5Notifications.so.5 knotifications-5.26.0_1
 libKF5TextWidgets.so.5 ktextwidgets-5.26.0_1
 libKF5Wallet.so.5 kwallet-5.26.0_1
+libkwalletbackend5.so.5 kwallet-5.34.0_1
```
Comment (asergi) [inline common/shlibs]: `'s/5.34.0/5.35.0/g'`

**Comment by yopito** [comment]
@asergi : thanks for pointing this.
I've just fixed all intermediates commits to `common/shlibs`.

**Comment by asergi** [comment]
Please rebase.

5.36 is out. Can you update it? Else I'll just merge.

**Comment by yopito** [comment]
Ready for review and merge.

My packaging status (5.36):
* (native) build is fine with x86_64-glibc for all these packages
* cross build is fine for x86_64-musl for all these packages
* armv7hf or aarch64 (glibc or musl) cross build status: **72**/75 packages are OK.  
* broken cross-build packages (flagged with `broken` keyword) are:
    * `plasma-framework` : "target `Qt5::Gui_GLESv2` not found"
    * `krunner` : depend on `plasma-framework`
    * `kdelibs4support` : broken because of #6658
* `gpgme` : enhance with Qt support (or this will break Travis build on `libqgpgme.so`)

@ebfe and @asergi : I'm really waiting for review and merge if OK to everyone . On my side, continuous updating this PR is really time consuming, more than "just" update. Please feeback.

**Comment by ebfe** [comment]
Thanks! And sorry for the delay...

**Pull request closed without merging**